docs/Dockerfile

```
@@ -0,0 +1,89 @@
+# --- Build stage --------------------------------------------------------------
+FROM base AS build
+
+# build-essential + libpq-dev/libyaml-dev to compile native gems if a precompiled
+# variant isn't used; git so the gem's gemspec (`git ls-files`) resolves the path
+# dependency; unzip for the Bun installer.
+RUN apt-get update -qq && \
+    apt-get install --no-install-recommends -y \
+      build-essential curl git libpq-dev libyaml-dev pkg-config unzip
+
+# Bun runs the Tailwind CSS v4 CLI (bun run build:css).
+RUN curl -fsSL https://bun.sh/install | bash -s "bun-v${BUN_VERSION}"
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)